assemblies/psw-ce/src/main/resources/log4j2.xml

```diff
@@ -19,7 +19,7 @@
         <!-- ================================= -->
 
         <!-- A time/date based rolling appender -->
-        <RollingFile name="FILE" fileName="${sys:user.home}/.schemaWorkbench/workbench.log" filePattern="${sys:user.home}/.schemaWorkbench/workbench.log.%d{yyyy-MM-dd}" append="false" bufferedIO="true" immediateFlush="true">
+        <RollingFile name="FILE" fileName="${sys:user.home}/.schemaWorkbench/workbench.log" filePattern="${sys:user.home}/.schemaWorkbench/workbench.log.%i" append="false">
```

Comment:
I think we should remove append=false so that when PSW is re-started, the workbench.log isn't truncated.

Reply:
Also, I suggest we remove the OnStartupTriggeringPolicy since this is completely new log4jxml2 behavior.